docs/USER_GROUP_API.md

```diff
@@ -110,7 +110,7 @@ For example, introspection is not available, and the resolver logic is not used.
 
 ## Other Services
 
-The `systemd` project provides three other services implementing this
+The `systemd` project provides five other services implementing this
```

Comment:
Maybe say "several" given it's now many and more will surely come, so that only the list needs updating?